### ISSUE TITLE
aliceVision_meshDecimate 's result not good

### ISSUE THREAD

**Issue by csunking** [alicevision/AliceVision#809]
Left image is the original mesh, and right image is the result. The right image seems has inerratic holes which looks weird.   But I don't know if I give the wrong parameters (just input obj, output obj and simplificaitonFactor). 
![123](https://user-images.githubusercontent.com/6279897/84392994-a29e2380-ac2d-11ea-88a6-eaefb2ed3b7b.png)

Hope can get some help.

**Comment by natowi** [comment]
Take a look at [InstantMeshes](https://github.com/wjakob/instant-meshes)

**Comment by csunking** [comment]
@natowi ,  Thanks,  I will have a try.

**Comment by fabiencastan** [comment]
You should not decimate by more than twice. If you need more you need to add MeshFiltering and alternate filtering/decimate multiple times.

This needs to be integrated natively into the MeshDecimate node but currently it decimates without any filtering (without any modification of the vertices positions).

**Comment by csunking** [comment]
@natowi    InstantMeshes is better than aliceVision_meshDecimate (small holes), but still has holes and even I turned the checkbox of "Align to boundaries" on.
![InstantMeshes ](https://user-images.githubusercontent.com/6279897/84499939-2ddaf000-ace6-11ea-922c-353fbc09b31b.png)
After many times try, I found that my mesh data seems had been splited into parts of rect box. 
Now, my trouble is the exist programs (aliceVision_meshDecimate,  InstantMeshes ) can not rmap the newly generated obj mesh of texture.
